lib/manager/npm/extract/index.ts

```diff
@@ -213,6 +213,9 @@ export async function extractPackageFile(
       } else if (depName === 'yarn') {
         dep.datasource = npmId;
         dep.commitMessageTopic = 'Yarn';
+      } else if (depName === 'npm') {
+        dep.datasource = npmId;
+        dep.commitMessageTopic = 'npm';
```

Comment:
Not needed

Reply:
I'm sorry, I'm not sure I understand. Could you explain what's unnecessary?

Reply:
See new suggestion

Reply:
Ah, gotcha. Thanks!